psp/scripts/train_model.py

```diff
@@ -94,6 +94,13 @@ def _eval_model(model: PvSiteModel, dataloader: "DataLoader[Sample]") -> None:
     default=False,
     show_default=True,
 )
+@click.option(
+    "--no_infer",
+    is_flag=True,
```

Comment:
This is good! One that can be slightly easier is to use `store_action=True` which does what this `is_flag`,`default=False` thing does.